### PR TITLE
Remove unneeded helpers from parseTools.js. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: src/library.js [hunk 1]
```diff
@@ -27,12 +27,12 @@ LibraryManager.library = {
...
   getTempRet0: function() {
-    return {{{ makeGetTempRet0() }}};
+    return getTempRet0();
   },
 
   setTempRet0__sig: 'vi',
-  setTempRet0: function($i) {
-    {{{ makeSetTempRet0('$i') }}};
+  setTempRet0: function(val) {
+    setTempRet0(val);
```
Comment (kripken) [inline src/library.js:35]:
```suggestion
    setTempRet0(val | 0);
```
With this, this is not NFC, unless you verified all the callers.

File: src/library.js [hunk 2]
```diff
@@ -27,12 +27,12 @@ LibraryManager.library = {
 
   getTempRet0__sig: 'i',
   getTempRet0: function() {
-    return {{{ makeGetTempRet0() }}};
+    return getTempRet0();
   },
```
Comment (kripken) [inline src/library.js:31]: this basically makes `_getTempRet0` forward the call to `getTempRet0`. Could we just mark this method as `$getTempRet0` (unprefixed), and include the body here? (moving it from `support.js`)

But this PR looks safe and good as it is, and that can be left for later.

**Review by kripken — COMMENTED** [review]
lgtm % comments

File: src/library.js (hunk 1 above)
Comment (sbc100) [reply src/library.js:35]: Oh, I thought from our conversation about that the `| 0` stuff was no longer needed so could just be skipped.  I didn't do any kind of verification so I guess I need to restore this.

File: src/library.js (hunk 2 above)
Comment (sbc100) [reply src/library.js:31]: Yes, indeed, my understanding that the library functions `_getTempRet0` and `_getTempRet1` are nothing but wrappers about the runtime functions.. in order to allow the runtime functions to be used by native code (native code can't link against runtime functions directly).

I think removing the runtime versions from support.js might cause issue with external code that expects to be able to call the unprefixed versions.

File: src/library.js (hunk 1 above)
Comment (sbc100) [reply src/library.js:35]: My understanding is that all callers of these `_` prefixed versions should be native code .. there are no callers that I know of in JS that use the `_` prefixed versions.. they all call the support.js versions AFAICT.

Comment (kripken) [reply src/library.js:35]: (I don't see any myself.)

File: src/library.js (hunk 2 above)
Comment (kripken) [reply src/library.js:31]: Yes, sounds like merging the prefixed and unprefixed versions would be nontrivial.

**Pull request merged**